app/views/guests/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1> <%=@guest.name%> </h1>
+
+<h2> Rooms </h2>
+<ul>
+    <% @guest.rooms.each do |room| %>
+        <li>
+            Suite: <%= room.suite %>
+            <br>
+            Rate: <%= room.rate %>
+            <br>
+            Hotel: <%= room.hotel.name %>
+        </li>
+        <br>
+    <% end %>
+</ul>
+
+<div id="add_room">
+    <%= form_with url:"/guests/#{@guest.id}", method: :patch do |form| %>
```

Comment:
Spoke about this in the controller and why this would be a POST over a PATCH. The path for this form should be `"/guests/#{@guest.id}/rooms"`